### PR TITLE
Partial changelog update for beta6 release

### PR DESCRIPTION
**Pull request by Jinming-Hu**
# Pull Request Checklist

Please leverage this checklist as a reminder to address commonly occurring feedback when submitting a pull request to make sure your PR can be reviewed quickly:

See the detailed list in the [contributing guide](https://github.com/Azure/azure-sdk-for-cpp/blob/master/CONTRIBUTING.md#pull-requests).

- [x] [C++ Guidelines](https://azure.github.io/azure-sdk/cpp_introduction.html)
- [x] Doxygen docs
- [x] Unit tests
- [x] No unwanted commits/changes
- [x] Descriptive title/description
  - [x] PR is single purpose
  - [x] Related issue listed
- [x] Comments in source
- [x] No typos
- [x] Update changelog
- [x] Not work-in-progress
- [x] External references or docs updated
- [x] Self review of PR done
- [x] Any breaking changes?

### CODE REVIEW

File: sdk/storage/azure-storage-blobs/CHANGELOG.md [hunk 1]
```diff
@@ -71,6 +72,12 @@
 - `DownloadBlobResult::Content-Range` is changed to an `Azure::Core::Http::Range`, an extra field `BlobSize` is added.
 - Remove `Undelete` from `BlobContainerClient`.
 - `BlobRetentionPolicy::Enabled` is renamed to `BlobRetentionPolicy::IsEnabled`, `BlobStaticWebsite::Enabled` is renamed to `BlobStaticWebsite::IsEnabled`.
+- Change type for metadta to case-insensitive `std::map`.
```
Comment (ahsonkhan) [inline sdk/storage/azure-storage-blobs/CHANGELOG.md:75]:
```suggestion
- Change type for metadata to case-insensitive `std::map`.
```

File: sdk/storage/azure-storage-blobs/CHANGELOG.md [hunk 2]
```diff
@@ -71,6 +72,12 @@
...
+- Rename `SharedKeyCredential` to `StorageSharedKeyCredential`.
+- Move `StoragePerRetryPolicy` and `StorageSharedKeyPolicy` to `Details` namespace.
```
Comment (ahsonkhan) [inline sdk/storage/azure-storage-blobs/CHANGELOG.md]: Frame these changelog entries from the perspective of the end user by highlighting what's relevant to them.
```suggestion
- Make `StoragePerRetryPolicy` and `StorageSharedKeyPolicy` private by moving them to the`Details` namespace.
```

File: sdk/storage/azure-storage-files-datalake/CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,9 @@
 
 ## 12.0.0-beta.6 (Unreleased)
 
+### New Features
+- Support setting blob sas permission with a raw string.
```
Comment (ahsonkhan) [inline sdk/storage/azure-storage-files-datalake/CHANGELOG.md]: Is this accurate and do we need to repeat it for each service?

File: sdk/storage/azure-storage-files-datalake/CHANGELOG.md [hunk 4]
```diff
@@ -2,6 +2,9 @@
 
 ## 12.0.0-beta.6 (Unreleased)
 
+### New Features
+- Support setting DataLake sas permission with a raw string.
```
Comment (ahsonkhan) [inline sdk/storage/azure-storage-files-datalake/CHANGELOG.md:6]: Add a new line after a heading and capitalize `sas`, here/elsewhere.
```suggestion
### New Features

- Support setting DataLake SAS permission with a raw string.
```

File: sdk/storage/azure-storage-files-datalake/CHANGELOG.md [hunk 5]
```diff
@@ -2,6 +2,9 @@
 
 ## 12.0.0-beta.6 (Unreleased)
```
Comment (ahsonkhan) [inline sdk/storage/azure-storage-files-datalake/CHANGELOG.md:3]: Date change here and `azure-storage-files-shares/CHANGELOG.md` is missing.

Comment (Jinming-Hu) [reply sdk/storage/azure-storage-files-datalake/CHANGELOG.md:3]: @katmsft will do datalake service and file service and then update the date

File: sdk/storage/azure-storage-files-datalake/CHANGELOG.md (hunk 3 above)
Comment (Jinming-Hu) [reply sdk/storage/azure-storage-files-datalake/CHANGELOG.md]: updated

File: sdk/storage/azure-storage-files-shares/CHANGELOG.md [hunk 6]
```diff
@@ -26,6 +26,7 @@
 ### New Features
 
 - Added support for `CreateIfNotExists` for Share and Directory clients, and `DeleteIfExists` for Share, Directory and File clients.
+- Support setting file sas permission with a raw string.
```
Comment (ahsonkhan) [inline sdk/storage/azure-storage-files-shares/CHANGELOG.md:29]: Should this be `files shares` or is `file` more accurate and grammatically correct?
```suggestion
- Support setting file SAS permission with a raw string.
```

File: sdk/storage/azure-storage-files-datalake/CHANGELOG.md [hunk 7]
```diff
@@ -2,6 +2,10 @@
...
 
+### New Features
+
+- Support setting DataLake sas permission with a raw string.
```
Comment (ahsonkhan) [inline sdk/storage/azure-storage-files-datalake/CHANGELOG.md:7]:
```suggestion
- Support setting DataLake SAS permission with a raw string.
```

File: sdk/storage/azure-storage-common/CHANGELOG.md [hunk 8]
```diff
@@ -1,16 +1,18 @@
 # Release History
 
-## 12.0.0-beta.6 (Unreleased)
+## 12.0.0-beta.6 (2020-01-14)
 
 ### New Features
 
 - Add new type `ContentHash`.
+- Add definition of `Metadata`.
+- Support setting account sas permission with a raw string.
```
Comment (ahsonkhan) [inline sdk/storage/azure-storage-common/CHANGELOG.md:9]:
```suggestion
- Support setting account SAS permission with a raw string.
```

**Review by katmsft — APPROVED** [review]

File: sdk/storage/azure-storage-blobs/CHANGELOG.md [hunk 9]
```diff
@@ -9,6 +9,7 @@
 - New API: `PageBlobClient::GetPageRangesDiff` and `PageBlobClient::GetManagedDiskPageRangesDiff`.
 - Add `CreateBlobContainer`, `DeleteBlobContainer`, `UndeleteBlobContainer` into `BlobServiceClient`.
 - Add `DeleteBlob` into `BlobContainerClient`.
+- Support setting blob sas permission with a raw string.
```
Comment (ahsonkhan) [inline sdk/storage/azure-storage-blobs/CHANGELOG.md:12]: nit - Missed this one:
```suggestion
	- Support setting blob SAS permission with a raw string.
```

**Pull request merged**